database/index.js

```diff
@@ -0,0 +1,118 @@
+const mongoose = require('mongoose');
+const IP = '172.31.34.129';
+const PORT = '27017'
+const seedData = require('./data/seed_data.js');
+
+mongoose.connect(`mongodb://localhost/breed`, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useCreateIndex: true
+});
+
+var db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error:'));
+
+
+db.once('open', ()=> {
+  console.log(`mongoose connected`)
+  //seed(seedData.breedInfoArray)
+})
+
+let breedSchema = mongoose.Schema({
+  id: Number,
+
+  about: mongoose.Schema({
+    breedName: String,
+    availableForAdoption: Number,
+    imageUrl: String,
+    summary: String,
+    otherNames: String
+  }),
+
+  stats: mongoose.Schema({
+    energyLevel: Number,
+    exerciseRequirements: Number,
+    playfullness: Number,
+    affectionLevel: Number,
+    friendlinessToDogs: Number,
+    friendlinessToOtherPets: Number,
+    friendlinessToStrangers: Number,
+    watchfullness: Number,
+    easeOfTraining: Number,
+    groomingRequirements: Number,
+    heatSensitivity: Number,
+    vocality: Number
+  }),
+
+  specs: mongoose.Schema({
+    type: String,
+    weightMin: Number,
+    weightMax: Number,
+    heightMin: Number,
+    heightMax: Number,
+    family: String,
+    areaOfOrigin: String,
+    dateOfOrigin: String,
+    otherNames: String
+  }),
+
+  history: String,
+  temprament: String,
+  upkeep: String,
+
+  health: mongoose.Schema({
+    majorConcerns: String,
+    minorConcerns: String,
+    occaisonallySeen: String,
+    suggestedTests: String,
+    lifeSpan: String,
+  }),
+    disclaimer: String
+
+})
+
+let Breed = mongoose.model('Breed', breedSchema);
+
+//checks if db has any documents
+let dbCheck = () => {
+  let findPromise = Breed.find({id:2}).exec();
+  return findPromise;
+}
+
+
+let seed = (breedInfoArray) => {
+  //check if there is an existing breed DB
+  dbCheck()
```

Comment:
good call on checking for data. don't know if this is a good suggestion from me, but could make it update the data if it already exists. that way if you decide to change your seed/re-seed for some reason it could still work. I did this using upsert